Makefile

```
@@ -112,21 +112,18 @@ install/kubectl-gadget: kubectl-gadget-$(GOHOSTOS)-$(GOHOSTARCH)
 	mkdir -p ~/.local/bin/
 	cp kubectl-gadget-$(GOHOSTOS)-$(GOHOSTARCH) ~/.local/bin/kubectl-gadget
 
-# gadget BCC container image
-.PHONY: gadget-default-container
-gadget-default-container:
-	docker buildx build -t $(CONTAINER_REPO):$(IMAGE_TAG) -f Dockerfiles/gadget-default.Dockerfile \
-		--build-arg ENABLE_BTFGEN=$(ENABLE_BTFGEN) .
+GADGET_CONTAINERS = \
+	gadget-default-container \
+	gadget-core-container
+
+gadget-container-all: $(GADGET_CONTAINERS)
 
-# gadget CO-RE container image
-.PHONY: gadget-core-container
-gadget-core-container:
-	docker buildx build -t $(CONTAINER_REPO):$(IMAGE_TAG) -f Dockerfiles/gadget-core.Dockerfile \
+gadget-%-container:
+	docker buildx build -t $(CONTAINER_REPO):$(IMAGE_TAG)$(if $(findstring core,$*),-core,) -f Dockerfiles/gadget-$*.Dockerfile \
 		--build-arg ENABLE_BTFGEN=$(ENABLE_BTFGEN) .
 
-.PHONY: push-gadget-container
-push-gadget-container:
-	docker push $(CONTAINER_REPO):$(IMAGE_TAG)
+push-gadget-%-container:
+	docker push $(CONTAINER_REPO):$(IMAGE_TAG)$(if $(findstring core,$(branch)),-core,)
```

make push-gadget-core-container is pushing the wrong image for me, perhaps the following suggestion fixes it?

```
$ make push-gadget-core-container 
docker push 192.168.1.150:5000/gadget:francis-makefile-containers
```


```suggestion
	docker push $(CONTAINER_REPO):$(IMAGE_TAG)$(if $(findstring core,$*),-core,)
```